src/DropdownBase/DropdownBase.js

```diff
@@ -307,6 +308,7 @@ class DropdownBase extends React.PureComponent {
           open: this._open,
           close: this._close,
           toggle: this._toggle,
+          isOpen,
```

Comment:
If open can be undefined:
```suggestion
          isOpen: Boolean(open),
```